packages/core/src/routes/session.ts

```diff
@@ -171,18 +198,28 @@ export default function sessionRoutes<T extends AnonymousRouter>(router: T, prov
     }),
     async (ctx, next) => {
       const { connectorId, code, state, redirectUri } = ctx.guard.body;
+      ctx.log = {
+        ...ctx.log,
```

Comment:
the spreading is not aligned with others?

Reply:
Updated. PTAL